ios/CodePush/CodePushPackage.m

```diff
@@ -231,18 +245,38 @@ + (void)downloadPackage:(NSDictionary *)updatePackage
                                                             }
                                                         }
                                                         
-                                                        if (isDiffUpdate && ![CodePushUpdateUtils verifyHashForDiffUpdate:newUpdateFolderPath
-                                                                                                             expectedHash:newUpdateHash
-                                                                                                                    error:&error]) {
-                                                            if (error) {
-                                                                failCallback(error);
+                                                        if(isDiffUpdate){
+                                                            CPLog(@"Applying diff update.");
+                                                        }else{
+                                                            CPLog(@"Applying full update.");
+                                                        }
+                                                        
+                                                        BOOL isSignatureVerificationEnabled = (publicKey != nil);
+                                                        if (isSignatureVerificationEnabled) {
+                                                            BOOL isSignatureValid = [CodePushUpdateUtils verifySignatureFor:newUpdateFolderPath
+                                                                                      withPublicKey:publicKey
+                                                                                              error:&error];
+                                                            if(!isSignatureValid){
```

Comment:
Same style issue, please use spaces (and please check all the places below)

Reply:
@max-mironov got it!